calico_kubernetes/calico_kubernetes.py

```diff
@@ -82,26 +80,25 @@ def create(self, namespace, pod_name, docker_id):
                          e.returncode, e.output, e)
             sys.exit(1)
 
+        # Give the policy agent a reference to the endpoint in the API
```

Comment:
Add a comment explaining that this must be done after configuring the default profile to avoid the race condition we talked about.